modules/programs/eww.nix

```diff
@@ -45,26 +107,73 @@ in {
     };
   };
 
-  config = mkIf cfg.enable {
-    home.packages = [ cfg.package ];
-    xdg.configFile."eww".source = cfg.configDir;
-
-    programs.bash.initExtra = mkIf cfg.enableBashIntegration ''
-      if [[ $TERM != "dumb" ]]; then
-        eval "$(${ewwCmd} shell-completions --shell bash)"
-      fi
-    '';
-
-    programs.zsh.initExtra = mkIf cfg.enableZshIntegration ''
-      if [[ $TERM != "dumb" ]]; then
-        eval "$(${ewwCmd} shell-completions --shell zsh)"
-      fi
-    '';
-
-    programs.fish.interactiveShellInit = mkIf cfg.enableFishIntegration ''
-      if test "$TERM" != "dumb"
-        eval "$(${ewwCmd} shell-completions --shell fish)"
-      end
-    '';
-  };
+  config = mkIf cfg.enable (mkMerge [
+    {
+      assertions = let
+        message = "You can have either yuck & scss or configDir enabled";
+      in [
+        {
+          assertion = !(cfg.scss.enable && cfg.configDir.enable);
+          inherit message;
+        }
+        {
+          assertion = !(cfg.yuck.enable && cfg.configDir.enable);
+          inherit message;
+        }
+      ];
+
+      home.packages = [ cfg.package ];
+
+      programs.bash.initExtra = let ewwCmd = "${cfg.package}/bin/eww";
```

Comment:
Why are you re-defining the command here? It's already in-scope on line 9.

Reply:
Ops I forgot to remove that.